### PR TITLE
Add php8 support and migrate to rubix-ml

### PR DESCRIPTION
**Pull request by ChristophWurst**
* Add php8 linters and tests
* Drop the abandoned and disputed php-ml and jump over to rubix-ml
  + Our own Dataset could die because rubix has its own
  + Training works almost identical, prection as well
  + All CLI commands were tested and work
  + Prediction with an old php-ml model fails but is handled properly just like when the app is installed for the very first time. This triggers a warning in the logs but nothing more. When the first background job builds a model, all is fine again and real classification happens
  + The model performance dropped from ~95% to ~80% because I have to tweak and optimize the new algorithm
* Add a new seed command that fills the db with random data, useful when no real data is available but training results are naturally rather meaningless and poor
  + Added CI workflow to train on a seeded database. It's more or less an automated smoke test

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-versions: [ 7.2, 7.3, 7.4 ]
+        php-versions: [ 7.2, 7.3, 7.4, 8.0 ]
```
Comment (blizzz) [inline .github/workflows/lint.yml:22]: kill 7.2?

File: lib/Command/Seed.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+		parent::__construct("suspiciouslogin:seed");
+		$this->seeder = $seeder;
+
+		$this->addOption(
+			'v6',
+			null,
+			InputOption::VALUE_NONE,
+			"train with IPv6 data"
+		);
+	}
```
Comment (blizzz) [inline lib/Command/Seed.php:51]: Maybe add a description and state it is random data?

Comment (ChristophWurst) [reply lib/Command/Seed.php:51]: Reminds me that I wanted to error out when someone runs this without debug mode …

**Review by blizzz — APPROVED** [review]
:elephant:

**Comment by akondas** [comment]
" abandoned and disputed php-ml", thanks :clap:

**Pull request merged**